modules/elements-color-picker/assets/js/editor/commands/start.js

```diff
@@ -0,0 +1,21 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class Start extends CommandBase {
+	apply( args ) {
+		elementor.$previewContents.find( 'body' ).addClass( 'elementor-editor__ui-state__color-picker' );
+
+		this.component.currentPicker = {
+			container: args.container,
+			control: args.control,
+			initialColor: args.container.getSetting( args.control ),
+		};
+
+		elementor.$previewContents.on( 'mouseenter.color-picker', '.elementor-element', ( e ) => {
+			$e.run( 'elements-color-picker/show-swatches', { id: e.currentTarget.dataset.id } );
+		} );
+
+		elementor.$previewWrapper.on( 'mouseleave.color-picker', () => {
+			$e.run( 'elements-color-picker/end', { value: this.component.currentPicker.initialColor } );
```

Comment:
```suggestion
			$e.run( 'elements-color-picker/exit-preview', { value: this.component.currentPicker.initialColor } );
```

Reply:
`exit-preview` will fire as soon as the mouse leaves the palette anyway, so it will be pointless to change this line to achieve this.
This line was intended to stop the picking state when the preview losses focus, so the user won't have to explicitly do an action to stop it. Sometimes they decide to leave the picking and just add another widget or something.